src/components/manage/Controlpanels/AddonsControlpanel.jsx

```diff
@@ -0,0 +1,384 @@
+/**
+ * Users controlpanel container.
+ * @module components/manage/Controlpanels/AddonsControlpanel
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Link } from 'react-router';
+import { Portal } from 'react-portal';
+import { Accordion, Button, Divider, Header, Icon, Label, Message, Segment } from 'semantic-ui-react';
+import jwtDecode from 'jwt-decode';
+import {
+  FormattedMessage,
+  defineMessages,
+  injectIntl,
+  intlShape,
+} from 'react-intl';
+
+import { installAddon, listAddons, uninstallAddon } from '../../../actions';
+import { getBaseUrl } from '../../../helpers';
+import { Toolbar } from '../../../components';
+
+const messages = defineMessages({
+  activateAndDeactivate: {
+    id: 'Activate and deactivate',
+    defaultMessage: 'Activate and deactivate add-ons in the lists below.',
+  },
+  addAddons: {
+    id: 'Add Addons',
+    defaultMessage: 'To make new add-ons show up here, add them to your buildout configuration, run buildout, and restart the server process. For detailed instructions see',
+  },
+  addonsSettings: {
+    id: 'Add-ons Settings',
+    defaultMessage: 'Add-ons Settings',
+  },
+  available: {
+    id: 'Available',
+    defaultMessage: 'Available',
+  },
+  back: {
+    id: 'Back',
+    defaultMessage: 'Back',
+  },
+  installed: {
+    id: 'Installed',
+    defaultMessage: 'Installed',
+  },
+  installedVersion: {
+    id: 'Installed Version',
+    defaultMessage: 'Installed Version',
+  },
+  update: {
+    id: 'Update',
+    defaultMessage: 'Update',
+  },
+  updatesAvailable: {
+    id: 'Updates available',
+    defaultMessage: 'Updates available',
+  },
+  updateInstalledAddons: {
+    id: 'Update installed addons:',
+    defaultMessage: 'Update installed addons:',
+  },
+  uninstall: {
+    id: 'Uninstall',
+    defaultMessage: 'Uninstall',
+  },
+});
+
+@injectIntl
+@connect(
+  (state, props) => ({
+    installedAddons: state.addons.installedAddons,
+    availableAddons: state.addons.availableAddons,
+    upgradableAddons: state.addons.upgradableAddons,
+    activeIndex: state.activeIndex,
```

Comment:
This is a variable which is defined in the local state of your component, so you don't need this line here. The `state` variable in this method refers to the global Redux store.